python/tvm/hybrid/calls.py

```diff
@@ -0,0 +1,92 @@
+"""Intrinsics of TVM-Python Hybrid Script for Python compilation time
+semantic support."""
+
+import ast
+from .. import api as _api
+from .. import expr as _expr
+from .. import make as _make
+from ..container import Array
+from ..ir_pass import Equal
+from ..stmt import For
+from .util import _internal_assert
+
+#pylint: disable=redefined-builtin
+
+LOOP_INTRIN = {
+    'range'    : For.Serial,
+    'unroll'   : For.Unrolled,
+    'parallel' : For.Parallel,
+    'vectorize': For.Vectorized,
+}
+
+def _range(func_id, args):
+    """Handling TVM loop types"""
+    n = len(args)
+    if n == 1:
+        low, ext = _api.const(0, dtype='int32'), args[0]
```

Comment:
Can we have const as args[0] here?

Reply:
Implicitly, inductive variables have dtype int32, so I do not know what you mean here.

Reply:
I mean, user may want have other dtype inductive variables, like int64